tests/cli/clean/test_hk_case_bundle_files.py

```diff
@@ -0,0 +1,163 @@
+import logging
+import datetime as dt
+
+from cgmodels.cg.constants import Pipeline
+
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.models.cg_config import CGConfig
+from click.testing import CliRunner
+from tests.store_helpers import StoreHelpers
+
+from cg.cli.clean import hk_case_bundle_files, PIPELINE_PROTECTED_TAGS
+
+from cg.store import Store
+
+from cg.store import models
+from housekeeper.store import models as hk_models
+
+
+def test_date_days_ago_zero_days():
+    # GIVEN
+
+    # WHEN calculating days ago 0 days ago
+    result: dt.datetime = get_date_days_ago(0)
+
+    # THEN result should be equal to today
+    assert isinstance(result, dt.datetime)
+    assert result.date() == dt.datetime.now().date()
+
+
+def test_date_days_ago_one_days():
+    # GIVEN we know what date it was yesterday
+    days_ago = 1
+    yesterday = dt.datetime.now() - dt.timedelta(days=days_ago)
+
+    # WHEN calculating days ago 1 days ago
+    result: dt.datetime = get_date_days_ago(days_ago)
+
+    # THEN result should be equal to yesterday
+    assert isinstance(result, dt.datetime)
+    assert result.date() == yesterday.date()
+
+
+def get_date_days_ago(days_ago: int) -> dt.datetime:
+    return dt.datetime.now() - dt.timedelta(days=days_ago)
```

Comment:
Why is this here and not imported from code that will be used when actually executing or added as a test fixture for reuse by other tests?

Reply:
Moved